src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-utils.c

```diff
@@ -277,6 +278,18 @@ long xclmgmt_hot_reset(struct xclmgmt_dev *lro, bool force)
 #else
 		xclmgmt_reset_pci(lro);
 #endif
+		/*
+		 * For U50 built with 2RP flow. PLP Gate is closed after
+		 * pci hot reset. Any access to PLP IPs will hangs the host.
+		 * E.g. read/write BAR2 (DMA BAR). 
+		 * To be able to open PLP gate after hot reset, the first 64
+		 * bytes of config space has to be restored. 
+		 * In the meanwhile, XRT expects firewall trip instead of
+		 * hard hang if there is an unexpected access of non-exist
+		 * IPs. (E.g. Invalid access from a active VM)
```

Comment:
So, I'm confused. If user access PLP IP before gate opens, do we see firewall trip or host hang?